features/upgrade/routing/upgrade.feature

```diff
@@ -208,3 +208,85 @@ Feature: Routing and DNS related scenarios
     Then the step should succeed
     And the output should contain "Hello-OpenShift web-server-rc"
     """
+
+
+  @upgrade-prepare
+  @users=upuser1,upuser2
+  @admin
+  Scenario: Unidling a route work without user intervention - prepare
+    Given I switch to cluster admin pseudo user
```

Comment:
`Given I switch to the first user` should work.

Reply:
done